docs/reference/contrib/modeladmin/create_edit_delete_views.rst

```diff
@@ -150,3 +150,12 @@ See the following part of the docs to find out more:
 
 See the following part of the docs to find out more:
 :ref:`modeladmin_overriding_views`
+
+.. _modeladmin_exclude_fields:
+
+-----------------------------------
+``ModelAdmin.exclude_fields``
+-----------------------------------
+
+**Expected value**: An array of Model fields to be excluded from Create/Edit pages.
```

Comment:
I would describe the expected value as a list of 'field names' rather than Model fields. I know it's very unlikely that anyone would try to provide an actual list of Model field, but it can't hurt to be clear. Also, I think it would be better to follow the other examples in the docs and explain the **Expected value** in one paragrah, then how and where the value is used in a separate paragraph below.